chord_metadata_service/mohpackets/schemas/base.py

```diff
@@ -99,6 +98,16 @@
 #                                      #
 ########################################
 
+
+class DateInterval(Schema):
+    day_interval: Optional[int] = Field(
+        None, description="number of days since first diagnosis"
+    )
+    month_interval: Optional[int] = Field(
```

Comment:
Should month_interval be required? There should always be a month_interval, coming out of clinical ETL, but maybe it would be good to enforce it here as well?

Reply:
That would also remove the option month_interval = null, so that month_interval always requires a value regardless. How does ETL handle null value for date now?

Reply:
It enforces whatever the schema says. If you had the date as nullable, it'd be nullable.

Reply:
ok, I can make the changes